libqtile/resources/default_config.py

```diff
@@ -186,6 +194,10 @@
 # When using the Wayland backend, this can be used to configure input devices.
 wl_input_rules = None
 
+# This can be used instead of `screens` to split a physical monitor into multiple
+# 'screens': http://docs.qtile.org/en/stable/manual/config/screens.html#fake-screens
```

Comment:
One for another PR but we should add much more commentary in our default config.

Reply:
100% agree